### PR TITLE
feat: wishlist page

### PR DESCRIPTION
**Pull request by harshitadaswani**
added wishlist page

### CODE REVIEW

File: stylesheets/base.css [hunk 1]
```diff
@@ -55,3 +55,7 @@
 .relative {
     position: relative;
 }
+
+.color-red {
+    color: red;
```
Comment (Namrata-J) [inline stylesheets/base.css:60]: try to use color variables, if possible!!
the rest of the code looks good!!
you can merge it

**Review by Namrata-J — APPROVED** [review]

File: stylesheets/base.css [hunk 2]
```diff
@@ -55,3 +55,7 @@
...
+}
```
Comment (Namrata-J) [inline stylesheets/base.css:61]: you can use much more shorter name, for an example say, clr-r or simply c-red!!
just a suggestion the rest of the code is good!

**Review by Kothalkarpriya — APPROVED** [review]
Good to merge

File: stylesheets/base.css (hunk 2 above)
Comment (harshitadaswani) [reply stylesheets/base.css:61]: okay done

**Pull request merged**